marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/CreateAndUpdateModelTest.java

```diff
@@ -230,6 +339,23 @@ private void assertIndexDeployment() {
         });
     }
 
+    private void getIndexes(Boolean isIndexDeleted) {
```

Comment:
This method name doesn't indicate what it's doing - i.e. a "get" method is expected to return something. Instead, I think you want two methods that will each use the method I mention below:

```
private void assertDateTimeIndexExists()
private void assertDateTimeIndexDoesntExist()
```

Those are the exact things you want to prove, so having methods with those names makes good sense.